MANIFEST.in

```diff
@@ -1,4 +1,10 @@
-include setup.py MANIFEST MANIFEST.in
+include LICENSE.rst
+include README.rst
+include CHANGES.rst
+include setup.py
+include setup.cfg
+include MANIFEST
+include MANIFEST.in
 recursive-include pyraf *
-recursive-include docs pyraf_tutorial.pdf pyraf_tutorial.ps.gz pyraf_install_src.ps pyraf_install_src.pdf
```

Comment:
I moved the tutorial to our our [web page](https://iraf-community.github.io/pyraf.html), since it is a read-only document without source (if you have the sources, that would be great!).

The remaining two documents (ECL mode and ipyton notes) are IMO not really helpful for the user; and then it is better to leave them out completely. I am even not sure whether they are current.
 
And for API docs, my feeling is that the current docstring don't give a helpful (for the user) documentation yet, resp. I was too lazy to setup a sphinx build.

Reply:
Ah, ok. Thanks for the clarification!